com.unity.netcode.gameobjects/Editor/HiddenScriptEditor.cs

```diff
@@ -21,14 +20,16 @@ public override void OnInspectorGUI()
         }
     }
 
+#if UNITY_UNET_PRESENT
```

Comment:
this macro is not defined in the editor assembly so this disables this for all versions as currently written.